README.md

```diff
@@ -22,61 +25,75 @@ This package requires Python 3.5+
 
-* Settings file, e.g.
-  ```yaml
-  # located in ~/.chaostoolkit/settings.yaml
-  auths:
-    chaosapi.proofdock.io:
-      type: bearer
-      value: <your_api_token_here>
-  controls:
-    proofdock:
-      provider:
-        arguments:
-          api_url: https://chaosapi.proofdock.io/
-        module: pdchaoskit.controls
-        type: python
-  ```
-* You may override the settings file with the environment variable ``PROOFDOCK_API_TOKEN`` which overrides the token from your settings file.
 
-The Proofdock Chaos CLI will first try to load the configuration from the settings file. If no configuration is provided, it will try to load it from the environment variables. Please check the [usage command][proofdock_chaoskit_configure] to see how to set up the settings file with the Proofdock Chaos CLI.
+## Commands
 
-### Putting it all together
+The Proofdock Chaos CLI introduces new commands:
+- `chaos configure`
+- `chaos experiment`
 
-Here is a full example to configure the Chaos CLI, run an experiment and list those experiments.
+and extends existing `chaos run` command with new options.
 
-1. Configure the settings file with the API token that you have gathered from the Proofdock Chaos Engineering Platform.
-   ```bash
-   chaos configure --token <your-api-token>
-   ```
-   More configuration options are vailable. For more options check out the CLI ``--help`` command itself or have a look at the [docs][proofdock_chaoskit_configure].
+### Run
+You use the `chaos run` command to run an experiment and upload its results to the Proofdock cloud.
+
+``` bash
+chaos run [OPTIONS] SOURCE
+```
 
+**Options**
+``` bash
+  --description TEXT  (new)  Additional description of the experiment run.
+  --journal-path TEXT        Path where to save the journal from the execution.
+  --dry                      Run the experiment without executing activities.
+  --no-upload         (new)  Do not upload the experiment and results after running.
+  --no-validation            Do not validate the experiment before running.
+```
 
-2. Run an experiment and send its results to the Proofdock cloud.
-    ```bash
-    chaos run /path/to/experiment.yml
-    ```
-   You may use several options like providing descriptions to your experiment run. For more options check out the CLI ``--help`` command itself or have a look at the [docs][proofdock_chaoskit_run].
+### Configure
+You use the `chaos configure` command to set the API token in a settings file.
 
+``` bash
+chaos configure --token <API token>
+```
 
-3. Retrieve and list experiments from the Proofdock cloud.
-    ```bash
-    chaos experiment list
-    ```
-    For more options check out the CLI ``--help`` command itself or have a look at the [docs][proofdock_chaoskit_list].
+**Options**
+``` bash
+  --token TEXT            Token value
+  --default-api-url TEXT  Default API
+```
 
+### List
+You use ``experiment list`` command to list all experiments in your project.
```

Comment:
The same: Wiederholungsfehler :)

Reply:
I will not fix it as this would break the convention we decide on earlier